test/unit/test_actions.py

```diff
@@ -484,22 +484,29 @@ def test_record_state_xml(self, xpatch):
         with open(os.path.join(self.test_run_dir, state_path)) as state_data:
             assert state_data.read() == test_state_data_xml
 
+    def invoke_import_state_files(self, path, expected_success = True, **extra_action_params):
+        output = self.invoke_action('import-state-files',
+                                    file_path_pattern=os.path.join(path, '*1.state.cfg'),
+                                    format="c-style",
+                                    target_format="c-style",
+                                    merge=False,
+                                    **extra_action_params)
+        states = None
+        if expected_success:
```

Comment:
With `expected_success == False` this does not really do anything - `ActionError` exceptions are caught inside XMNR code.